manifests/cli.pp

```diff
@@ -22,6 +22,9 @@
     Class['jenkins::service']
       -> Class['jenkins::cli']
         -> Anchor['jenkins::end']
+    $require = Class['::jenkins::service']
```

Comment:
Isn't this redundant from the 2 lines above? I think you can just leave `$require` out entirely.

Reply:
true. purged it.